src/Traits/Livewire/CreatesDocuments.php

```diff
@@ -178,7 +225,11 @@ protected function createDocumentFromItems(
                 }
 
                 if ($isDownload) {
-                    $downloadIds[] = $media->getKey();
+                    if ($media->getKey() && is_int($media->getKey())) {
```

Comment:
it can be dangerous to check if key is an integer. What if the media model uses UUIDs as keys?